### PR TITLE
Conditionalise CentOS content

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/8921

@juliogonzalez not sure if i got those commands right. Please check?

### CODE REVIEW

**Comment by Loquacity** [comment]
> Extensive ifdefs are not good. Better create dedicated files and duplicate some parts or paragraphs. Linus Torvalds sometimes rants about these topics (iirc, keywords to search for are "driver" and "ifdef").

I absolutely agree with you, but we don't have a good technical solution for this yet. I didn't want to leave the issue hanging while we decide.

File: modules/client-configuration/pages/clients-centos.adoc [hunk 1]
```diff
@@ -34,6 +40,28 @@ spacewalk-common-channels -u admin -p `secret` -a x86_64 'centos7-updates'
...
++
+----
+zypper in spacewalk-utils
+----
+. Add the {centos} base, updates, and client channels using the [command]``centos6-uyuni-client`` script:
++
+----
+centos6-uyuni-client -u admin -p `secret` -a x86_64 'centos7'
+centos6-uyuni-client -u admin -p `secret` -a x86_64 'centos7-updates'
+----
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-centos.adoc]: What you need is the `spacewalk-common-channels` script, and call it like this:
```
spacewalk-common-channels centos7 centos7-updates centos7-uyuni-client
```
For CentOS7.

And same for CentOS6, just replacing `7` with `6`

File: modules/client-configuration/pages/clients-centos.adoc [hunk 2]
```diff
@@ -34,6 +40,28 @@ spacewalk-common-channels -u admin -p `secret` -a x86_64 'centos7-updates'
...
+ifeval::[{uyuni-content} == true]
+
+.Procedure: Adding Channels and Repositories
+
+. At the command prompt on the {productname} Server, as root, install the [package]``spacewalk-utils`` package:
++
+----
+zypper in spacewalk-utils
+----
+. Add the {centos} base, updates, and client channels using the [command]``centos6-uyuni-client`` script:
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-centos.adoc]: `spacewalk-common-channels`

File: modules/client-configuration/pages/clients-centos.adoc [hunk 3]
```diff
@@ -34,6 +40,28 @@ spacewalk-common-channels -u admin -p `secret` -a x86_64 'centos7-updates'
 spacewalk-common-channels -u admin -p `secret` -a x86_64 'centos7-uyuni-client-x86_64'
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-centos.adoc]: This should be enough.
```
spacewalk-common-channels centos7 centos7-updates centos7-uyuni-client
```

But I am not sure if this is what we do. On SUSE Manager content, should we use the RES7 client tools?

**Comment by juliogonzalez** [comment]
@Loquacity, you can close this one, as I reworked the doc at https://github.com/SUSE/doc-susemanager/pull/759, including the conditionals.

**Comment by juliogonzalez** [comment]
I will close it, so it doesn't get merged by accident.

**Pull request closed without merging**